Installation.md

```diff
@@ -13,6 +13,9 @@ Those dependencies are listed below:
 
 Note that only Homebrew package is maintained by the git-extras developers directly.
 Other packages are maintained by the distribution's packagers or third-party volunteers.
+And their versions are usually not up-to-date.
```

Comment:
```suggestion
Other packages are maintained by the distribution's packagers or third-party volunteers and their versions are not always up-to-date.
```

Sounds a bit more gentle :)

Reply:
@SuperSandro2000 
Thank you for your suggestion.
I have updated the text according to your advice.